content/en/synthetics/private_locations.md

```diff
@@ -16,7 +16,7 @@ further_reading:
 ---
 
 <div class="alert alert-warning">
-This feature is in public beta and available for API Tests only.
+This feature is in public beta for API Tests. The equivalent feature for Browser Tests is in private beta, if you would like to test it, [reach out to Datadog support][1].
```

Comment:
since you are in a html bow you need the html way 👓 to write links

```suggestion
This feature is in public beta for API Tests. The equivalent feature for Browser Tests is in private beta. <a href="https://docs.datadoghq.com/help/">Reach out to Datadog support</a> if you would like to test it.
```